### PR TITLE
fix(build): Fix build fail by increasing dex heap size

### PR DESCRIPTION
**Pull request by Bambooin**

### CODE REVIEW

**Comment by EmmanuelMess** [comment]
Is build failing? I've built recently...

**Comment by Bambooin** [comment]
In my machine, it is failed recently.
PS: 
oracle java 1.8.0_162
geadle 4.4.1
macOS 10.13.2
android studio 3.0

**Comment by EmmanuelMess** [comment]
Have you changed the build variant to playDebug?

**Comment by VishalNehra** [comment]
A fresh project import might actually fail.

**Comment by Bambooin** [comment]
Before TranceLove's sftp branch merging, I can build playRelease. Then my build will hang on dex gradle task for a long time, then gradle exception will throw and my build fail. After increasing the dex heap size, my build will success quickly.

File: app/build.gradle [hunk 1]
```diff
@@ -86,6 +87,7 @@ dependencies {
     implementation 'com.android.support:appcompat-v7:26.1.0'
     implementation 'com.android.support:palette-v7:26.1.0'
     implementation 'com.android.support:cardview-v7:26.1.0'
+    implementation 'com.android.support:multidex:1.0.2'
```
Comment (EmmanuelMess) [inline app/build.gradle]: Why this line?

Comment (Bambooin) [reply app/build.gradle]: Yes, it is not necessary.

I add it as for this:
multiDexEnabled true

**Comment by Bambooin** [comment]
It's the error log:

`AGPBI: {"kind":"error","text":"Uncaught translation error: java.util.concurrent.ExecutionException: java.lang.OutOfMemoryError: GC overhead limit exceeded","sources":[{}]}
AGPBI: {"kind":"error","text":"Uncaught translation error: java.util.concurrent.ExecutionException: java.lang.OutOfMemoryError: GC overhead limit exceeded","sources":[{}]}
AGPBI: {"kind":"error","text":"2 errors; aborting","sources":[{}]}

:app:transformClassesWithDexForPlayRelease FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:transformClassesWithDexForPlayRelease'.
> com.android.build.api.transform.TransformException: com.android.ide.common.process.ProcessException: java.util.concurrent.ExecutionException: com.android.ide.common.process.ProcessException: Return code 1 for dex process
`

File: app/build.gradle (hunk 1 above)
Comment (EmmanuelMess) [reply app/build.gradle]: Remove and I'll pull

Comment (Bambooin) [reply app/build.gradle]: I already remove and rebase it in c69742a.

Comment (EmmanuelMess) [reply app/build.gradle]: Kudos for rebasing

**Pull request merged**